machines/flyctl/fly-machine-run.html.md

```diff
@@ -3,33 +3,32 @@ title: Run a new Machine
 objective: Run a new Fly Machine with flyctl
 layout: docs
 nav: machines
-order: 5
+redirect_from: /docs/machines/run/
 ---
 
-The [`fly machine run` command](/docs/flyctl/machine-run/) is a tool to configure, build, and run a new Machine in a single guided interaction.
+The [`fly machine run`](/docs/flyctl/machine-run/) command is a tool to configure, build, and start a new Machine in one line.
 
-Use `fly machine run` to include Machines built from more than one single Docker image in a Fly App, or to run a one-off or temporary Machine.
-
-Fly Launch features like `fly deploy`, `fly status`, and `fly scale` don't apply to Machines created with `fly machine run`, unless you [add metadata to indicate otherwise](/docs/machines/run#add-metadata-to-the-machine).
-
-Here's what `fly machine run` does for you:
-* Checks with the platform for the org and app you've specified, if any, and if needed, guides you through naming a new app
-* Creates a Fly App, if applicable
-* Gets, or builds, a Docker image to make the Machine from
-* Creates the Machine with some default config, plus config you pass to it with flags
-* Starts the Machine
-* Waits for the Machine to start before declaring success (or failure)
+Many, but not all, [Machine configuration](/docs/machines/api-machines-resource/#the-machine-config-object-properties/) options are available to the `fly machine run` command through flags. The available flags are listed in the flyctl help and on the [`fly machine run` reference page](/docs/flyctl/machine-run/).
 
+<div class="note icon">
```

Comment:
> I think this would work well as a sub-section rather than a note. Maybe called "Other machine commands" or similar.

Leaving this for now, since I think the other machine commands belong with the fly scale note and I don't know what to call the section.

> Line 53: the "even if you're not using fly deploy confused me at first in the context of the doc. 

I changed this; better?